bundler.d/assets.rb

```diff
@@ -16,3 +16,4 @@
 gem 'spice-html5-rails', '~> 0.1.5'
 gem 'quiet_assets', '~> 1.0'
 gem 'jquery_pwstrength_bootstrap_4', :require => 'jquery_pwstrength_bootstrap'
+gem 'ipaddrjs-rails', '~> 1.1.1'
```

Comment:
This needs to be packaged, @timogoebel let me know if you want me to do it or you prefer to do it. The README in https://github.com/theforeman/foreman-packaging/tree/rpm/develop explains how to do this, it's very simple (run gem2rpm , update comps/ and rel-eng/tito.props and submit the PR :smile: )